src/main/java/com/alevel/hw10/Main.java

```diff
@@ -0,0 +1,22 @@
+package com.alevel.hw10;
+
+import com.alevel.hw10.models.*;
+
+public class Main {
+    public static void main(String[] args) {
+        PassengerCar passengerCar = new PassengerCar(Series.AA, 123, Color.BLACK);
+
+        Truck truck = new Truck(Series.AX, 124, Color.GREEN);
+
+        passengerCar.addPassenger();
+        passengerCar.addPassenger();
+        passengerCar.addPassenger();
+        passengerCar.addPassenger();
+
+        System.out.println(passengerCar.getTraffic());
+        System.out.println(truck.getTraffic());
+
+        Recovery car = new PassengerCar(Series.AX, 242, Color.RED);
```

Comment:
Good, but I want to see the implementation: `Recovery recovery = new Recovery()`

Reply:
Added second option